package-lock.json

```diff
@@ -2854,9 +2853,10 @@
       }
     },
     "node_modules/fetch-mock": {
-      "version": "9.11.0",
-      "resolved": "https://registry.npmjs.org/fetch-mock/-/fetch-mock-9.11.0.tgz",
-      "integrity": "sha512-PG1XUv+x7iag5p/iNHD4/jdpxL9FtVSqRMUQhPab4hVDt80T1MH5ehzVrL2IdXO9Q2iBggArFvPqjUbHFuI58Q==",
+      "name": "@gr2m/fetch-mock",
+      "version": "9.11.0-pull-request-644.1",
+      "resolved": "https://registry.npmjs.org/@gr2m/fetch-mock/-/fetch-mock-9.11.0-pull-request-644.1.tgz",
+      "integrity": "sha512-gTp6RCHzlOXS1qRb0APfuyz48Lw/JFPa4uiar+kEgL1STsDwth75HJZ4x30tBlXMJXV8XDTDzJ2Hz9w3RWiHJA==",
```

Comment:
This section wasn't updated in the previous PR and was causing the need for `node-fetch`.
npm was still installing the unpatched version of `fetch-mock` even if it was updated in the `package.json`.

You have to explicitly install it via the CLI for it to update

Reply:
ah got it, thank you!